agreements/project-contribution/fiscal-sponsorship-amendment.md

```diff
@@ -0,0 +1,61 @@
+# Amendment to Fiscal Sponsorship Agreement
+
+**Project:** `_____________________________________`  
+**Original Agreement Date:** `__________________________`  
+**Amendment Effective Date:** `__________________________`  
+
+Pursuant to the terms of the Fiscal Sponsorship Agreement and the incorporated [Terms and Conditions][], the following amendments are made:
+
+---
+
+## Addition of a Project Signatory
+
+The following individual is added as a Project Signatory. By signing below, they agree to be bound by all terms of the Fiscal Sponsorship Agreement and the Commonhaus Foundation Terms and Conditions.
+
+**New Signatory:**  
+`Name:      ⎣_____________________________________________⎦`  
+`Email:     ⎣_____________________________________________⎦`  
+
+`Signature: ⎣_____________________________________________⎦`  
+`Date:      ⎣_____________________________________________⎦`  
+
+---
+
+## Resignation of a Project Signatory
+
+The following individual has provided written notice of resignation and is hereby removed as a Signatory under the Agreement:
+
+**Resigning Signatory:**  
+`Name:      ⎣_____________________________________________⎦`  
+`Email:     ⎣_____________________________________________⎦`  
```

Comment:
Should there be an `effective date` here? Just trying to think through the effect of resigning with notice e.g is the agreement updated upon notice of intent to resign or the date of effective resignation (assuming the two are different)

Reply:
Amendment Effective Date is at the top.

Reply:
Ok.

For some reason I can't mark the thread as resolved :/